### PR TITLE
feat(upload file): init cli upload file command PE-6636

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

**Review by dtfiedler — APPROVED** [review]

File: src/cli/utils.ts [hunk 1]
```diff
@@ -168,37 +170,52 @@ const tokenToDevGatewayMap: Record<TokenType, string> = {
...
+}
+
 export function getUploadFolderOptions(options: UploadFolderOptions): {
   folderPath: string;
   indexFile: string | undefined;
   fallbackFile: string | undefined;
   disableManifest: boolean;
   maxConcurrentUploads: number;
 } {
+  console.log('options', options);
```
Comment (dtfiedler) [inline src/cli/utils.ts:214]: nit: looks like this was used for debugging

Comment (fedellen) [reply src/cli/utils.ts]: nice catch c3ad1e7

**Pull request merged**